packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -72,16 +70,17 @@ const LabelContainer = styled.div`
     justify-content: flex-start;
     overflow: hidden;
 
-    &:hover {
+    :hover {
```

Comment:
nit:
![Screenshot 2022-08-16 at 16 18 09](https://user-images.githubusercontent.com/33235762/184902657-44bb0148-cf8c-477e-89bf-cfd2e080804a.png)

![Screenshot 2022-08-16 at 16 16 21](https://user-images.githubusercontent.com/33235762/184902424-43718570-2b7f-42cd-b355-8ba7e1428a68.png)
Separator, different color or shadow for address part would be nice. It is hard to read the address on hover.

Reply:
b406e14399ceff4177e5ab91e78e4e775ad2fd70 added a pipe with lower opacity:

![Screenshot 2022-08-16 at 18 05 07](https://user-images.githubusercontent.com/42465546/184926561-e4efc5e4-f3fe-4935-a61c-883646f7ff1c.png)


Let me know if that's fine and I will merge it.

Reply:
Better not put any typical characters. Consider `|`, `•` or space + shadow.

Reply:
Please, make the space to the right of the separator a bit bigger and perhaps `0.3` opacity, otherwise much better!

Reply:
I reduced the opacity to 0.25 and both increased both margins to 6px. The left margin only looks bigger with the ellipsis, I don't know what to do about that, but most users will see it this way:

![Screenshot 2022-08-17 at 11 28 03](https://user-images.githubusercontent.com/42465546/185085657-5fbf5031-2467-44c9-b394-6e557abc7eac.png)

Reply:
Much better, cool!